processor/k8sattributesprocessor/internal/kube/informer.go

```diff
@@ -80,57 +150,47 @@ func informerWatchFuncWithSelectors(client kubernetes.Interface, namespace strin
 	}
 }
 
-// newKubeSystemSharedInformer watches only kube-system namespace
-func newKubeSystemSharedInformer(
-	client kubernetes.Interface,
-) cache.SharedInformer {
-	informer := cache.NewSharedInformer(
-		&cache.ListWatch{
-			ListFunc: func(opts metav1.ListOptions) (runtime.Object, error) {
-				opts.FieldSelector = fields.OneTermEqualSelector("metadata.name", kubeSystemNamespace).String()
-				return client.CoreV1().Namespaces().List(context.Background(), opts)
-			},
-			WatchFunc: func(opts metav1.ListOptions) (watch.Interface, error) {
-				opts.FieldSelector = fields.OneTermEqualSelector("metadata.name", kubeSystemNamespace).String()
-				return client.CoreV1().Namespaces().Watch(context.Background(), opts)
-			},
-		},
-		&api_v1.Namespace{},
-		watchSyncPeriod,
-	)
-	return informer
-}
-
 func newNamespaceSharedInformer(
 	client kubernetes.Interface,
-) cache.SharedInformer {
+	fs fields.Selector,
+	stopCh chan struct{},
```

Comment:
Small suggestions, I think `stopCh <-chan struct{}` is more accurate

Reply:
Good point, done.